Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -69,20 +75,31 @@ fun ReplyInboxScreen(
         }
     }
 
+    var selectedEmailIds by remember { mutableStateOf(setOf<Long>()) }
```

Comment:
sounds like you would like to remembersave this

Reply:
Done